scripts/west_commands/runners/nrfjprog.py

```diff
@@ -62,7 +62,13 @@ def name(cls):
 
     @classmethod
     def capabilities(cls):
-        return RunnerCaps(commands={'flash'}, erase=True)
+        return RunnerCaps(commands={'flash'}, dev_id=True, erase=True)
+
+    @classmethod
+    def dev_id_help(cls) -> str:
+        return '''Device identifier. Use it to select the J-Link Serial Number
```

Comment:
What this supposed to say "J-Link"? Or is it the serial number of the nRF board?

Reply:
All of our boards use a J-Link debugger, and the serial number is associated with the J-Link OB chip, so I think it's a good description.